CMakeLists.txt

```
@@ -44,6 +44,20 @@ endif (NOT BUILD_STATIC)
 ## Retrieve git revision (at configure time)
 find_package (Git)
 
+## make format
+message (STATUS "Looking for clang-format...")
+find_program (CLANG_FORMAT clang-format)
+
+if (CLANG_FORMAT)
+  message (STATUS "Looking for clang-format... ${CLANG_FORMAT}")
+  add_custom_target(format /usr/bin/clang-format "-i" "./base/*.c" "./gmp/*.c"
+                    "./osp/*.c" "./util/*.c" "./base/*.h" "./gmp/*.h" "./osp/*.h"
+                    "./util/*.h")
```

Specify WORKING_DIRECTORY 
```suggestion
                    "./util/*.h" WORKING_DIRECTORY "../")
```